lib/ember-cli/app.rb

```diff
@@ -95,10 +95,14 @@ def ember_path
 
     private
 
-    delegate :match_version?, :non_production?, to: Helpers
+    delegate :match_version?, to: Helpers
     delegate :configuration, to: EmberCLI
     delegate :tee_path, :npm_path, :bundler_path, to: :configuration
 
+    def non_production?
+      !Rails.env.production?
```

Comment:
Relying on `Rails.env.production?` is an anti-pattern and totally now gonna work for any production-like environment with a different name. For instance lots of people actually create a separate environment called `staging` by copying `production.rb` into `staging.rb` and deploing it. This change ensures that none of this gonna work anymore. This is the main reason `Helpers.non_production?` exists in the first place.